src/nectarchain/user_scripts/ggrolleron/gain_PhotoStat_computation.py

```diff
@@ -124,10 +125,20 @@ def main(
     str_extractor_kwargs = CtapipeExtractor.get_extractor_kwargs_str(
         args.extractor_kwargs
     )
-    path = DataManagement.find_SPE_HHV(
+    # path = DataManagement.find_SPE_HHV(
+    #    run_number=args.HHV_run_number,
+    #    method=args.method,
+    #    str_extractor_kwargs=str_extractor_kwargs,
+    # )
+    # path = DataManagement.find_SPE_nominal(
+    #    run_number=args.HHV_run_number,
+    #    method=args.method,
+    #    str_extractor_kwargs=str_extractor_kwargs,
+    # )
+    path = DataManagement.find_SPE_nominal(
         run_number=args.HHV_run_number,
-        method=args.method,
-        str_extractor_kwargs=str_extractor_kwargs,
+        method="GlobalPeakWindowSum",
+        str_extractor_kwargs=f"window_width_{8}_window_shift_4",
```

Comment:
I would guess the f-string is not needed here ?

Reply:
True it's a mistake